addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -387,6 +386,11 @@ def _compute_accrual_count(self):
         for leave_type in self:
             leave_type.accrual_count = mapped_data.get(leave_type.id, 0)
 
+    @api.onchange('employee_requests')
```

Comment:
Is an `onchange()` required here? Is it not working with a compute stored field? :innocent:

Reply:
Not sure... Making the field compute and stored would make it readonly unless an inverse method is existing (and I don't think there is a field to assign a value to in the inverse method in this case).
The purpose of this onchange method is to make sure that there isn't a record that requires approval from Time Off Officer if employee requests are not allowed: https://www.awesomescreenshot.com/video/7145122. Since the approval field disappears from the form based on the employee requests being allowed or not, there is a risk of having the allocation_validation_type set to officer while the employee_requests is set to no without the onchange method.